js/plugins/evaluators/src/types.ts

```diff
@@ -22,3 +31,66 @@ export enum GenkitMetric {
   DEEP_EQUAL = 'DEEP_EQUAL',
   JSONATA = 'JSONATA',
 }
+
+export interface BaseGenkitMetricConfig {
+  type: GenkitMetric;
+  statusOverrideFn?: (score: Score) => EvalStatusEnum;
```

Comment:
@pavelgj I made this override method to use the score and get a new status. should we make this even more generic and just pass in the full `datapoint`?

This is part of the plugin API so should be easy to fix.... I think?